src/gui/optionsdialog.h

```diff
@@ -103,19 +103,45 @@ private slots:
     void on_addWatchedFolderButton_clicked();
     void on_editWatchedFolderButton_clicked();
     void on_removeWatchedFolderButton_clicked();
-    void on_registerDNSBtn_clicked();
     void setLocale(const QString &localeStr);
+
+#ifndef DISABLE_WEBUI
     void webUIHttpsCertChanged(const Path &path);
     void webUIHttpsKeyChanged(const Path &path);
+    void on_registerDNSBtn_clicked();
+#endif
 
 private:
     void showEvent(QShowEvent *e) override;
 
     // Methods
-    void saveOptions();
-    void loadOptions();
-    void initializeLanguageCombo();
+    void saveOptions() const;
+
+    void loadBehaviorTabOptions();
```

Comment:
For optionsdialog.cpp/.h, it is mostly moving code into its related function and some coding style clean up.

Reply:
> For optionsdialog.cpp/.h, it is mostly moving code into its related function and some coding style clean up.

I hope you didn't hide anything behind all this. I really can't review all these changes. So if there is something significant among them, it would be nice if you would point it out exactly.

Reply:
Not significant but worth pointing out.
There was this problem: https://github.com/qbittorrent/qBittorrent/pull/17392#issuecomment-1193244685
And is fixed here: https://github.com/qbittorrent/qBittorrent/blob/27a07e20f555ab0437e48e867ed4d9d8f76bf79a/src/gui/optionsdialog.cpp#L560-L561
The issue was `setSelectedPath()` was invoked before `setMode()` so it was using the wrong (default) mode.

Other than that is just moving code around and reordering them (according to UI layout).